tests/WP_SQLite_Driver_Metadata_Tests.php

```diff
@@ -61,7 +61,7 @@ public function testInformationSchemaTables() {
 				'CREATE_TIME'     => $result[0]->CREATE_TIME,
 				'UPDATE_TIME'     => null,
 				'CHECK_TIME'      => null,
-				'TABLE_COLLATION' => 'utf8mb4_general_ci',
+				'TABLE_COLLATION' => 'utf8mb4_0900_ai_ci',
```

Comment:
Why this collation?

Reply:
It's the default in MySQL 8+. See also the last commit separately for some more context. The upside is more correct MySQL compatibility, the downside that such exports won't work on older MySQL and MariaDB - but that's already the case with MySQL 8 as well. The default on MySQL 5 seems to be 'latin1_swedish_ci', which is not great.